### ISSUE TITLE
Stabilizer Design

### ISSUE THREAD

**Issue by ausshir** [UofSSpaceDesignTeam/balloon-software#8]
- Explore some new alternatives to compare with the counterweight arms

**Comment by bht822** [comment]
![image](https://cloud.githubusercontent.com/assets/20550516/17640856/c6524338-60be-11e6-98db-e6d0d2cd6537.png)
For a more stable design of the payload , I have a rather unconventional and crazy idea why not build something which looks similar to this pic. attached or this  a torpedo or high aerial bombs were used in WW 2 and were dropped from altitudes greater than 15000 ft and would free fall to the ground. 
We can modify the middle section to carry the equipment's plus have a camera on the nose section and modify the tail section to have a rudder for horizontal stability .
P.S: Its a cliche a brown guy talking about bombs and torpedos (just for giggles)  
![image](https://cloud.githubusercontent.com/assets/20550516/17640910/79d050de-60c0-11e6-9fdd-97ff3ab37a66.png)

**Comment by ottopasuuna** [comment]
How would this be oriented, pointing up/down, or sideways parrallel with the horizon?
Also, it may be a bit alarming to outsiders if they saw this falling to the ground :smile:

**Comment by bht822** [comment]
While going up the nose be pointing up and while coming down the nose be pointing down ,
Yes I though about people getting scared but we can label it usst hab or like something which says chill people it's just a payload device

**Comment by ottopasuuna** [comment]
Yeah, if we paint it bright green, and with the broken balloon half attached, it shouldn't be too much of a problem.
If it is nose up/down, how would this prevent rotation? Also, keep in mind that the amateur radio club also has a payload that attaches underneath ours.

**Comment by bht822** [comment]
The middle part would rotate on an axis so our cameras which are attached at the nose will be stable .i do have a landing system in mind as well which is nothing like flacon 6 but should work for our case

**Comment by ausshir** [comment]
Could you make a basic sketch? Sounds intriguing!

In general with a balloon pulling us up and the HAM payload hangs below so we don't have much say in the landing or ascent stability.

**Comment by bht822** [comment]
I see I'll make some sketches
